src/index.js

```diff
@@ -0,0 +1,64 @@
+import './style.css';
+
+function itemID() {
+  let previousID = JSON.parse(localStorage.getItem('itemID'));
+  const newID = previousID + 1;
+  if (previousID == null) {
+    previousID = 1;
+    localStorage.setItem('itemID', JSON.stringify(previousID));
+    return previousID;
+  }
+  localStorage.setItem('itemID', JSON.stringify(newID));
+  return newID;
+}
+
+// ADD NEW ITEM
+class AddItem {
+  constructor(ID, chore, complete) {
+    this.ID = ID;
+    this.chore = chore;
+    this.complete = complete;
+  }
+}
+const add = document.querySelector('#add');
+add.addEventListener('click', () => {
+  const ID = itemID();
+  const chore = document.querySelector('#todoInput').value;
+  const complete = document.querySelector('.completed');
+  const newItem = new AddItem(ID, chore, complete);
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md) for this project, " `Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].` ", please ensure the value of the `completed key` is boolean. Something like so: 
```
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
```
Currently the value is `null`, please kindly fix that.

Reply:
Thanks for your review! Consider changes made